cvc4/src/cvc4_sort.cpp

```diff
@@ -152,6 +152,10 @@ SortKind CVC4Sort::get_sort_kind() const
   {
     return DATATYPE;
   }
+  else if (sort.isConstructor())
+  {
+    return CONSTRUCTOR;
+  }
```

Comment:
What about `SELECTOR` and `TESTER`? It looks like we have the necessary functionality in the CVC4/cvc5 [API](https://github.com/cvc5/cvc5/blob/3dda54ba7e6952060766775c56969ab920430a8a/src/api/cvc4cpp.h#L386-L396).